### DISCUSSION TITLE
Generating 2D samples from 3D data

### DISCUSSION THREAD

**Discussion by hasansari** [Project-MONAI/MONAI#1641] [Q&A]
Hi, 

Is there an easy way to use RandSpatialCropSamplesd or something similar to generate 2D samples from a 3D image volume to be used in 2D Unet? This was easy to do in niftynet but can't figure it out in MONAI

Thanks a lot in advance!

**Comment by rijobro** [comment]
Hi, @wyli has just created [an issue](https://github.com/Project-MONAI/tutorials/issues/130) on our tutorial repository saying it would be nice to have a notebook showing this functionality. Check through the links in that issue to related questions. I think the most succinct answer is this one: https://github.com/Project-MONAI/MONAI/issues/808#issuecomment-663027024.
